tests/lib/peers/PeerInfo.test.ts

```diff
@@ -0,0 +1,22 @@
+import Polykey from "../../../src/lib/Polykey"
+import PeerInfo, { Address } from '../../../src/lib/peers/PeerInfo'
+
+describe('PeerInfo class', () => {
+  let peerInfoA: PeerInfo
+
+  beforeAll(async () => {
+    const mockPublicKey1 = '-----BEGIN PGP PUBLIC KEY BLOCK-----\nVersion: Keybase OpenPGP v2.0.82\nComment: https://keybase.io/crypto\n\nxsFNBF9Hd28BEADnJFlZHcXQu1LtuWmhEKvd9tTxXYKs+VK56aG72ucbW14yNv5R\nHZ+tqz9i4g1pQmGQ90yerma5jooPk4LsPzma4b/vgEGFWrUiJESJ3ggqHYKFtzxF\n4KHr9klcMBMGGN2FQOUIxC1sgfnemMAAHwp6mpK6Jg6ZjpNGILtkrKROLfmNqOEK\npHKAn/R6d/bZ6+G2Tti8/HcuiRMNevCY5KRFga5Zv0wDwq/u9+t/Wu1LwcJyJxQD\nBMaj3uootf6BgPr+7Jqsql21A+Fgm2C0ca6Msrjy8dhx7XLK8FHALLo/Qm8ZX/xX\nd2R631PKRF2J5UMKH70HDsUcQboLEC3GkuxwQciZx4wOqFfDDmzmk4GXms3a1oSz\nZGaWu275UBQ7Bl1OS46DdV//wYb7sG7lxzEe88vu5DAPO1P2qstNjRqEY4LqYt0A\njf+6ynVlXAsdwHVUmBwRKQuQTT4DdxLYOKlW9pmBnjltDR/6Z1wqw0m4a+ESCg4H\nGP2XV2wdcx0I50mQXt60iIwEzL46Sc5mermp0BeUBX+nN4cLxzmfVFItzBPvgils\na9iwX04PpB707KOe2eShhlEp07a9VQUTLYAfVZvrsjbpdY62T1+hop9et8tcFxdv\nKo0dQw3nLoGuxcEtxF6y3QhN6PCCCfxB5slaRCQ8ogiSxEnngATyGQfilQARAQAB\nzQ5Sb2JlcnQgPGVtYWlsPsLBdAQTAQoAHgUCX0d3bwIbLwMLCQcDFQoIAh4BAheA\nAxYCAQIZAQAKCRAvSvExafY2wv9vEACN38KxPxAsurLdSlcqaB9lDcKjYpEN58nh\nmmNA25mRGkjgBCZxEH3K6TpY5MIx7Z8x0BOtid/6pcHnBDhLnp91PG2msdaj4Ayx\ncqdWhGzipRkqDrNGv22cC51VNjiJPIlyh8NF4xlMEjoigUubb6/ppDyajbg+Wc08\n9qGfwSsBFFpGeNgLWNTgxtG++Ry3XNG5yaRJv/+Ro7iC531cidzXFLjRjCsRIX1/\npQnljxEr2tYUVibTODe7K2tdthLLVp4WhA4VPA7DWUZ21P/DHUxz56uHVDWkCSq+\npyHfbdkjnvCbQfFNVmFsNPa1iLXFS+Jwilfv34haVrhNwMjkd6QydGmWQwt79Ckk\nFIb+ubWenD5jrdwkllXWCeUAtfy6TNl2X/0rnrpF0zO5L3hnMl33alJMwp08n3WR\nxRVvyIbQ1XrFLKJE7bUDSoOUMJlpD7cUwiNSe1F/ptQ8HpJw0xY3DCrQ75TP85Jw\ncXgJYEXRzI3DlKRuayz8Miry+tX5HSHpbOPa4zUPit1xpgNeFWZmUqugNxqetqVx\nz3PQdnhI6mrcHyez254+6mapv/Ea5wuFcvn9tC/lp0Y/8dfhVETdwlPDxRGgglhB\nVFnbdll1LryCr7Nb2bj8P5+AEnrhSDAXgxXgf5bA/EMi4nEWZvNck0diBxZKaeh1\nvtIJ7zG91g==\n=gIGI\n-----END PGP PUBLIC KEY BLOCK-----\n'
```

Comment:
Can we use something to generate these as fixtures instead of saving these in the source?

Reply:
yeah can probably just use `kbpgp` actually. No need for these to be hardcoded

Reply:
done, no trace of mock, hardcoded pgp keys